tests/beaker_tests/netdev_stdlib/name_server_provider_defaults.rb

```diff
@@ -74,7 +74,7 @@
 
     # Expected exit_code is 0 since this is a vegas shell cmd.
     # Flag is set to true to check for absence of RegExp pattern in stdout.
-    cmd_str = UtilityLib.get_vshell_cmd('show running-config section name-server')
+    cmd_str = UtilityLib.get_vshell_cmd('show running-config | section name-server')
```

Comment:
Agreed. New diff is uploaded.

Thanks,
Edward
From: Mike Wiebe <notifications@github.com<mailto:notifications@github.com>>
Reply-To: cisco/cisco-network-puppet-module <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Thursday, December 10, 2015 at 6:33 AM
To: cisco/cisco-network-puppet-module <cisco-network-puppet-module@noreply.github.com<mailto:cisco-network-puppet-module@noreply.github.com>>
Cc: edpham <edpham@cisco.com<mailto:edpham@cisco.com>>
Subject: Re: [cisco-network-puppet-module] Fixing internal defect for name_server beaker test (#104)


In tests/beaker_tests/netdev_stdlib/name_server_provider_defaults.rb<https://github.com/cisco/cisco-network-puppet-module/pull/104#discussion_r47232168>:

> @@ -74,7 +74,7 @@
>
>      # Expected exit_code is 0 since this is a vegas shell cmd.
>      # Flag is set to true to check for absence of RegExp pattern in stdout.
> -    cmd_str = UtilityLib.get_vshell_cmd('show running-config section name-server')
> +    cmd_str = UtilityLib.get_vshell_cmd('show running-config | section name-server')


We should still remove it and replace it with an ensure => absent check similar to how radius_server tests. See the following.

https://github.com/cisco/cisco-network-puppet-module/blob/develop/tests/beaker_tests/radius_server/radius_server_provider_defaults.rb#L70

-
Reply to this email directly or view it on GitHub<https://github.com/cisco/cisco-network-puppet-module/pull/104/files#r47232168>.